orbs.rb

```diff
@@ -0,0 +1,23 @@
+require "sinatra"
+require 'sinatra/json'
+require "sinatra/reloader" if development?
+require "sequel"
+require_relative "db.rb"
+
+
+DB = Sequel.connect(adapter: 'mysql2', host: HOST, database: DBNAME, user: USERNAME, password: PASSWORD)
+get '/' do
+    DB.fetch("SELECT name,inet_ntoa(ip) as ip,water_uuid, elec_uuid, elec_rvid, water_rvid, relative_value FROM orbs o inner join relative_values rv on o.water_rvid=rv.id").map do |row|
+      content_type :json
+      output = {
```

Comment:
If we are only selecting these variables with the SQL, why do we need to restate them all in this hash?  Wouldn't `row` only contain these values?

Reply:
I don't really understand what you mean here.

Reply:
What is the difference between the object `output` and the object `row`?  I simply don't know.